src/integTest/groovy/com/bmuschko/gradle/docker/utils/DockerThreadContextClassLoaderIntegrationTest.groovy

```diff
@@ -223,7 +223,7 @@ class DockerThreadContextClassLoaderIntegrationTest extends AbstractIntegrationT
         when:
         def instance = null
 
-        def binds = ['/my/path': 'my/volume', '/other/path': '/other/volume']
+        def binds = ['/this/path': 'this/volume', '/that/path': '/that/volume', '/other/path': '/other/volume:ro']
```

Comment:
what is the difference between `'/this/path': 'this/volume', '/that/path': '/that/volume'`?

Reply:
None really I guess, I just appended to the test, with an additional value to test with the extra suffix.  There was a two element array without suffix , I could consolidate the test back to two elements, one with, one without a suffix.